Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contests-filters/ContestFilters.tsx

```diff
@@ -44,18 +47,25 @@ const ContestFilters = ({ onFilterClick }: IContestFiltersProps) => {
     const {
         state: { possibleFilters },
         actions: {
-            // toggleParam,
+            toggleParam,
             clearFilters,
             clearSorts,
         },
     } = useContests();
 
     const { actions: { clearBreadcrumb } } = useCategoriesBreadcrumbs();
 
-    // const handleSortClick = useCallback(
-    //     (sorting: ISort) => toggleParam(sorting),
-    //     [ toggleParam ],
-    // );
+    React.useEffect(() => {
```

Comment:
Why put React. infront of useEffect?

Reply:
habit from before, for me personally makes it easily readable if a specific hook is from React or if it is custom one.

Reply:
Let's be consistent with the rest of the project